src/module.ts

```diff
@@ -28,4 +28,25 @@ export class Module {
                 });
         });
     }
+
+    public webPost(url: string, data: any) {
+        return new Promise<string>((resolve, reject) => {
+            this.api.axios
+                .post(`https://${this.api.schoolId}.mobidziennik.pl${url}`, new URLSearchParams(data))
+                .then((el: AxiosResponse<any>) => {
+                    const text = el.data;
+                    if (
+                        text === 'Nie jestes zalogowany' ||
```

Comment:
Because we check this 2 (here and in webGet) can you extract it to another function?